protos/flyteidl/external/external_event.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package flyteidl.external;
+
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/external";
+
+import "flyteidl/core/execution.proto";
+import "flyteidl/core/identifier.proto";
+import "flyteidl/core/catalog.proto";
+import "flyteidl/core/workflow_closure.proto";
+import "flyteidl/core/compiler.proto";
+import "flyteidl/admin/event.proto";
+import "flyteidl/admin/common.proto";
+import "flyteidl/admin/execution.proto";
+import "flyteidl/admin/node_execution.proto";
+import "flyteidl/admin/task_execution.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/struct.proto";
+
+message ExecutionEventClosure {
+    flyteidl.core.ConnectionSet connections = 1;
```

Comment:
what's the purpose of this? I see it referenced as internal in the compiled workflow here: https://github.com/flyteorg/flyteidl/blob/091505aae78d935ae55e4d0e46deee5a1e79a22f/protos/flyteidl/core/compiler.proto#L28